<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_QtCalc</class>
 <widget class="QWidget" name="Widget_QtCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget_QtCalc</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>13</y>
     <width>221</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>A:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>B:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QDoubleSpinBox" name="Number_A">
                <property name="wrapping">
                 <bool>false</bool>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="keyboardTracking">
                 <bool>true</bool>
                </property>
                <property name="showGroupSeparator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="minimum">
                 <double>-100000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100000.000000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="Number_B">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="minimum">
                 <double>-10000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="stepType">
                 <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Resultado:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="Number_Resultado">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="pBut_Suma">
              <property name="text">
               <string>Suma</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pBut_Multiplica">
              <property name="text">
               <string>Multiplica</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pBut_Resta">
              <property name="text">
               <string>Resta</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pBut_Divide">
              <property name="text">
               <string>Divide</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Number_A</tabstop>
  <tabstop>Number_B</tabstop>
  <tabstop>pBut_Suma</tabstop>
  <tabstop>pBut_Resta</tabstop>
  <tabstop>pBut_Multiplica</tabstop>
  <tabstop>pBut_Divide</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Number_Resultado</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
